cmd/admin-trace.go

```diff
@@ -94,6 +101,22 @@ var adminTraceFlags = []cli.Flag{
 		Name:  "filter-size",
 		Usage: "filter size, use with filter (see UNITS)",
 	},
+	cli.StringFlag{
+		Name:  "rotate-trace-logs-size",
+		Usage: "maximum size of a rotated trace log file (see UNITS)",
+	},
+	cli.StringFlag{
+		Name:  "rotate-trace-logs-frequency",
+		Usage: "frequency, in terms of seconds, of trace log file rotations",
```

Comment:
Specify as duration, see [ParseDuration](https://github.com/allanrogerr/mc/blob/d2960d67cead1b795ae47042d6dbfcd9e8301829/cmd/duration.go#L94)

Reply:
This has been implemented and documented in the `mc admin trace` command (see top-most screenshot)